### DISCUSSION TITLE
How to setup zoned heating in HA using versatile thermostat

### DISCUSSION THREAD

**Discussion by asdkjasdfla** [jmcollin78/versatile_thermostat#302] [General]
I am trying to setup zoned heating in homeassistant. There are TRVs installed on radiators in zones and a Shelly switch controls a central gas heating boiler. Currently I am using Shelly switch via versatile thermostat and the smart TRVs turn on based on schedule. However apart from a single zone, whose temperature sensor feeds the versatile thermostat, being able to call for heat no other zone can call for heat. Wondering how can I make it work?

**Comment by jmcollin78** [comment]
Hello @asdkjasdfla ,

Your configuration is not what I recommend.

VTherm should control the TRV directly through the `over_valve` type if your valve are directly available (through a `number` entity) or through a `over_climate` VTherm.

Starting or stopping the boiler is not directly possible with VTherm but can be automated by catching the TRV value (if one is > 0 then start the boiler. If all TRV valve are open with 0 percent then turns off the boiler). Some people don't want to start / stop the boiler automatically and think this should be a human decision only. This depends on your configuration.

So with this configuration, you will be able to have one zone per Vtherm controlling one or more (up to 4) TRV.

Then you use the Scheduler to change the preset on the VTherm and not to change the TRV directly. If TRV are controlled by VTherm, it is better to not control them with another means.
